src/status_im2/contexts/activity_center/notification/mentions/view.cljs

```diff
@@ -0,0 +1,48 @@
+(ns status-im2.contexts.activity-center.notification.mentions.view
+  (:require [clojure.string :as string]
+            [i18n.i18n :as i18n]
+            [quo.components.animated.pressable :as animation]
+            [quo2.core :as quo]
+            [quo2.foundations.colors :as colors]
+            [status-im.utils.datetime :as datetime]
+            [status-im2.contexts.activity-center.notification.common.view :as common]
+            [status-im2.contexts.activity-center.notification.mentions.style :as style]
+            [utils.re-frame :as rf]))
+
+(defn message-body [message]
+  (let [parsed-text          (get-in message [:content :parsed-text])
+        parsed-text-children (:children (first parsed-text))]
+    (into [quo/text {:number-of-lines     2
+                     :style               style/tag-text
+                     :accessibility-label :activity-message-body
+                     :size                :paragraph-1}]
+          (map-indexed (fn [index {:keys [type literal]}]
```

Comment:
just wondering if `doall` is needed here, because we have deref inside

Reply:
@flexsurfer - It's not a deref. It's just a string which renders "@" with the user who got mentioned in the message.

Reply:
deref inside rf/sub

Reply:
I'm resolving this thread without changing anything because there's a call to `into` that realizes the lazy sequence produced by `map-indexed` into a persistent vector.

```clojure
(def lazy-foo (map println (range 5)))
(into [] lazy-foo)
;; => prints 0 1 2 3 4
```